### PR TITLE
Add --apns-mode option to specify which server to send to

### PR DESCRIPTION
**Pull request by nakajima**
We were always sending to development before. Once we start using the production servers, https://github.com/xmtp-labs/xmtp-inbox-ios/issues/58 should be fixed.

### CODE REVIEW

**Review by elisealix22 — APPROVED** [review]
👏 Amazing find 👏

File: pkg/options/options.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ type ApnsOptions struct {
 	KeyId                 string `long:"apns-key-id" env:"APNS_KEY_ID" description:"Key ID associated with APNS credentials"`
 	TeamId                string `long:"apns-team-id" env:"APNS_TEAM_ID" description:"APNS Team ID"`
 	Topic                 string `long:"apns-topic" env:"APNS_TOPIC" description:"Topic to be used on all messages"`
+	Mode                  string `long:"apns-mode" env:"APNS_MODE" description:"Which APNS servers to deliver to, development or production"`
```
Comment (neekolas) [inline pkg/options/options.go:15]: You can add `choice:"development" choice:"production"` to the struct tag to restrict the available options

**Review by neekolas — APPROVED** [review]
Great catch

File: pkg/options/options.go (hunk 1 above)
Comment (neekolas) [reply pkg/options/options.go:15]: I would also suggest adding a default. Otherwise this will fail to deploy, since nothing is configured right now.

**Review by neekolas — CHANGES_REQUESTED** [review]
We should add a default value before merging to prevent a failed deploy

File: pkg/options/options.go (hunk 1 above)
Comment (nakajima) [reply pkg/options/options.go]: Good calls, added both.

**Review by neekolas — APPROVED** [review]

**Pull request merged**